docs(AI): add AI development tools page and replace stale references (#4991)
patternfly/patternfly-org: packages/documentation-site/patternfly-docs/content/AI/rapid-prototyping/new-prototypes.md

Reviewer: ```suggestion
## Get started
```

The patternfly-react-seed ai_enabled branch comes pre-configured with:

- A working PatternFly React application structure with essential dependencies and build configuration.
- AI-friendly documentation that provides context about PatternFly components and best practices.
- Example components and patterns to reference.

### Set up the patternfly-react-seed

1. Clone or download the patternfly-react-seed repository using the ai_enabled branch.
2. Install dependencies according to [the project's README](https://github.com/patternfly/patternfly-react-seed/tree/ai_enabled?tab=readme-ov-file#patternfly-seed).

> 3. Install [AI helpers](https://github.com/patternfly/ai-helpers) marketplace plugins to give your AI tools additional context and skills for PatternFly development. See the [AI development tools](/ai/ai-development-tools) page for setup instructions.

Reviewer: ```suggestion
3. To give your AI tools additional context and skills for PatternFly development, install plugins from our [AI helpers](https://github.com/patternfly/ai-helpers) marketplace. For instructions on getting started, refer to our [AI development tools](/ai/ai-development-tools) guidance.
```
another note to double check the link before merging

4. Start the development server.
5. Begin prototyping with your AI development tools.

> ## How to prototype

Reviewer: ```suggestion
## How do I prototype?
```

> Once you've set up the patternfly-react-seed and installed the AI helpers plugins, follow these steps to begin prototyping:

Reviewer: ```suggestion
Once you've set up the patternfly-react-seed and installed the PatternFly AI helpers plugins, follow these steps to begin prototyping:
```
